File: dotnet/Hybrid.Automation.Framework/Hybrid.Automation.Framework/Pages/Homepage.cs
using Hybrid.Automation.Framework.Core;
using Hybrid.Automation.Framework.Core.Utils;
using Hybrid.Automation.Framework.Support;
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;

namespace Hybrid.Automation.Framework.Pages
{
    public class Homepage
    {
        private SharedDriver sharedDriver;

        [FindsBy(Using = "", How = How.CssSelector), CacheLookup] private IWebElement logo { get; set; }

        public Homepage(IWebDriver driver)
        {
            driver.Navigate().GoToUrl(EnvironmentConfiguration.GetEnvironment());
            AssertionHandlers.BrowserTitleShouldContain("");

            this.sharedDriver = (SharedDriver)driver;
        }
    }
}

File: selenium-java/src/test/java/automation/sites/GoogleTrader/HomePage.java
package automation.sites.GoogleTrader;

import automation.tests.SharedDriver;
import org.openqa.selenium.WebDriver;

import static automation.core.WebDriverExtensions.*;

public class HomePage {
    private final SharedDriver Driver;

    /*
        constructor
     */
    public HomePage(WebDriver driver) {
        navigateTo("http://www.google.com.ng/local/trader");
        this.Driver = (SharedDriver)driver;
    }
}
